metabrainz/templates/admin/stats/supporters.html

```diff
@@ -0,0 +1,27 @@
+{% extends 'admin/master.html' %}
+{% block body %}
+  <h1>Active supporters</h1>
+
+    <p>
+       All active users, in good standing with pledged amounts greater than zero:
```

Comment:
"users" here might be misleading. Maybe "supporters" as in the title above, or "commercial supporters"?

Reply:
Fixed. Even though the author is the only person to ever see this report. :)